src/components/TablePreview/TablePreview.tsx

```diff
@@ -38,15 +38,15 @@ const TablePreview: FC<Props> = ({}) => {
                             <img src={imgUrl} height="60"></img>
                         </td>
                         <td style={name.style}>{name.textValue}</td>
-                        <td>Phone Number</td>
+                        <td style={phone.style}>{phone.textValue}</td>
                     </tr>
                     <tr>
-                        <td>Title</td>
-                        <td>Email</td>
+                        <td style={title.style}>{title.textValue}</td>
+                        <td style={email.style}>{email.textValue} </td>
                     </tr>
                     <tr>
-                        <td>Company</td>
-                        <td>LinkedIn</td>
+                        <td style={company.style}>{company.textValue}</td>
+                        <td style={linkedin.style}>{linkedin.textValue}</td>
                     </tr>
```

Comment:
accessing the values from the objects to set the values in the HTML table very nice!